spec/models/organization_capacity_spec.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: organization_capacities
+#
+#  active_client_count :bigint
+#  capacity_limit      :integer
+#  name                :string
+#  vita_partner_id     :bigint           primary key
+#
+require 'rails_helper'
+
+describe OrganizationCapacity do
```

Comment:
I know we generally try to avoid too many looped tests like this, but this felt like a particularly good use case for it given that the intention of the test is to make sure that our view always appropriately tracks with the statuses we consider to be part of capacity.